plugins/woocommerce-blocks/assets/js/data/cart/notify-errors.ts

```diff
@@ -7,27 +7,50 @@ import { decodeEntities } from '@wordpress/html-entities';
 import { dispatch } from '@wordpress/data';
 
 /**
- * This function is used to notify the user of cart item errors/conflicts
+ * This function is used to normalize errors into an array of ApiErrorResponse objects.
+ */
+const filterValidErrors = ( errors: ApiErrorResponse | ApiErrorResponse[] ) => {
```

Comment:
I think a bit more info on what the two ternary cases are doing here would be good. e.g. when is `errors` _not_ going to be an `ApiErrorResponse`, and in that case why would we filtering it normalize the errors? Is it because `errors` can be an array of responses, but not all of them will be errors?

Reply:
It's because sometimes you receive a single error, other times you receive an array of errors. It depends on the route e.g. cart errors are an array, but an endpoint returning 402 is a single error. This just normalises it to an array and ensures each error has valid/expected properties. I can document in the doc block.

Reply:
Saying that, current usage seems to only use this for cart so maybe I can remove it. Ill try that and check tests continue to pass.